### PR TITLE
add base swagger kondo file for marketing to have something to render

### PR DESCRIPTION
**Pull request by AChelikani**
As we transition to kondo, the growth team is helping us setup a samsara.com/api-preview page that renders a swagger as an analog to the current samsara.com/api page. This is a initial base swagger for growth to use.

We likely do not want to have an automated pipeline setup that syncs backend kondo swagger with this one yet because we want to do staged launches as opposed to a continuous deployment of doc changes.

NOTE: we also need to clean up this description, but for now it is just acting as a dummy file to give growth something to render.

### CODE REVIEW

File: swagger-kondo.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (seward) [inline swagger-kondo.json]: Do we want `kondo` in the filename?

Comment (seward) [reply swagger-kondo.json]: Maybe `swagger-preview.json`?

File: swagger-kondo.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "swagger": "2.0",
+  "info": {
+    "title": "Samsara API",
```
Comment (silverspace) [inline swagger-kondo.json]: Nit: can we call this `Samsara API [PREVIEW]`?

Comment (AChelikani) [reply swagger-kondo.json]: Changed.

File: swagger-kondo.json (hunk 1 above)
Comment (AChelikani) [reply swagger-kondo.json]: That sounds better.

**Pull request merged**